src/Bot/commands/commandsReplies.ts

```diff
@@ -30,14 +30,15 @@ export const lookupReply = async (interaction: CommandInteraction) => {
     if (discordUserId) {
       // <@${discordUserId}> is used to create a link for the user profile
       await interaction.editReply({
-        content: `${usernameArg} on Discord is <@${discordUserId}>`,
+        content: `${usernameOption?.value} on Discord is <@${discordUserId}>`,
```

Comment:
It's not possible for `value` to be undefined because Discord requires the argument to be passed. If it was not passed because of a malfunction, then the userInfo query will throw an error.